modules/actor/actor-wfrp4e.js

```diff
@@ -1087,9 +1088,10 @@ export default class ActorWfrp4e extends Actor {
           testData.unofficialGrimoire.ingredientMode = html.find('[name="ingredientTypeSelected"]').val();
         }
         testData.malignantInfluence = html.find('[name="malignantInfluence"]').is(':checked');
+        testData.channelUntilSuccess = html.find('[name="channelUntilSuccess"]').is(':checked');
         testData.skillSelected = channellSkills[Number(html.find('[name="skillSelected"]').val())];
         testData.cardOptions = cardOptions;
-        if (options.channelUntilSuccess) {
+        if (testData.channelUntilSuccess) {
           return testData;
         } else {
           return new testData.rollClass(testData);
```

Comment:
So I probably want to ensure these callbacks **only** return Test objects, not their data.

If you had issues with duplicating the test object, the example I gave *should* work, I remember testing it

```js
let testData = duplicate(test); // Test is a test object, but duplication turns it into just its data
let testDuplicate = new test.rollClass(testData)
```

Reply:
Yes, I had problem with that, that was also one of the reasons why I had dedicated button for this instead of checkbox